src/plugins/salesforceChat.js

```diff
@@ -15,7 +15,7 @@ export const initializeSalesforceChat = () => {
   const license = store.getters['license/license']
 
   const initESW = function(gslbBaseURL) {
-    window.embedded_svc.settings.displayHelpButton = true // Or false
+    window.embedded_svc.settings.displayHelpButton = false // Or false
```

Comment:
```suggestion
    window.embedded_svc.settings.displayHelpButton = false
```
Probably no need for a comment here.